contracts/tvl-staking/DualRewardsDistributionRecipient.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+// Inheritance
+import "../common/Owned.sol";
+
+// https://docs.synthetix.io/contracts/source/contracts/rewardsdistributionrecipient
+abstract contract DualRewardsDistributionRecipient is Owned {
```

Comment:
This seems like a complete copy of the RewardsDistributionRecipient.sol, could we just not reuse it?

Reply:
The interface for the `notifyRewardAmount` function in `RewardsDistributionRecipient` is `notifyRewardAmount(uint256)`, but the `DualStakingRewards` contract needs `notifyRewardAmount(uint256,uint256)`

thats the only difference, I think I'll just remove the interface for `notifyRewardAmount` from `RewardsDistributionRecipient` so both `DualStakingRewards` and `StakingRewards` can reuse it

Reply:
Fixed in this commit: https://github.com/ribbon-finance/governance/pull/46/commits/80b8151e8e50c42c5d237cff161dc5adf7c9d9ac